roles/satellite-clone/defaults/main.yml

```diff
@@ -17,3 +17,4 @@ overwrite_etc_hosts: True
 puppet_version: 3
 check_networking_interfaces: True
 sp_features_table_name: smart_proxy_features
+selinux_pkgs: 
```

Comment:
@chris1984 this just needs to be a blank array, also `selinux_packages` would be a better name imo (I know I shortened it in the example)

Reply:
Fixed